docs/how-to-guides/develop-a-tool/add-category-and-tags-for-tool.md

```diff
@@ -0,0 +1,40 @@
+# Add Category and Tags for Tool
+At times, users require their tools to be readily accessible. To facilitate this, we've implemented the `category` and `tags` features. The category aids in compartmentalizing tools into distinct category folders, whereas the tags allow users to locate tools with corresponding tags.  
+Both category and tags assignments are optional. If a tool isn't assigned a category, it will be displayed in the root folder. Similarly, if no tags are assigned, the tags field will remain empty.
+
+## Prerequisites
+- Please ensure that your [Prompt flow for VS Code](https://marketplace.visualstudio.com/items?itemName=prompt-flow.prompt-flow) is updated to version 1.1.0 or a more recent version.
+
+## How to add category and tags for a tool
+Here we use [an existing tool](https://github.com/microsoft/promptflow/tree/main/examples/tools/tool-package-quickstart/my_tool_package/yamls/tool_with_file_path_input.yaml) as an example. If you wish to create your own tool, kindly refer to the [create and use tool package](create-and-use-tool-package.md#create-custom-tool-package) guide. You can add the _category_ and _tags_ fields in the tool's YAML like this:
```

Comment:
consistent format with before ? use `category` and ''tags' instead of _category_ and _tags_

Reply:
Changed, thanks.